.github/workflows/reusable-change-detection.yml

```diff
@@ -133,6 +133,7 @@ jobs:
       run: |
         echo "run-docs=true" >> "${GITHUB_OUTPUT}"
     - name: Get a list of the MSI installer-related files
+      if: github.event_name == 'pull_request'
```

Comment:
Oh, so the previous invocation of the action was conditional, and I forgot to copy it over. Sounds like this could use a code comment with the explanation of why this is necessary.